### PR TITLE
[JN-1523] preserving study sidebar during portal navigation

### PR DESCRIPTION
**Pull request by devonbush**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

Updates the portal dashboard to auto-select a study so that we can always have the study sidebar available.  Also changes some sidebar links so the study context is preserved while, e.g., navigating to edit the portal

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

1. go to https://localhost:3000/
2. select a portal
3. observe the full study sidebar now appears
4. click on Design -> Website
5. observe that the study context is preserved
6. go back to the portal dashboard
7. click on "Manage" links in the widgets.  Confirm study context is preserved

### CODE REVIEW

**Review by MatthewBemis — APPROVED** [review]
this pr is such a relief 🎉

File: ui-admin/src/navbar/StudySidebar.tsx [hunk 1]
```diff
@@ -120,6 +124,12 @@ export const StudySidebar = ({ study, portalList, portalShortcode }:
             className={sidebarNavLinkClasses} style={navStyleFunc}>Site Settings</NavLink>
         </li>
       </ul>}/>
+      <CollapsableMenu header={'Manage'} content={<ul className="list-unstyled">
```
Comment (connorlbark) [inline ui-admin/src/navbar/StudySidebar.tsx:127]: `Manage` sounds generic to me but I can't think of anything I'm super in love with... Maybe the section could be called `Study Staff` and the page is `Accounts`?

(As an aside, right now `Team members` has a different casing than the rest of the sidebar)

**Review by connorlbark — APPROVED** [review]
Nice!

File: ui-admin/src/navbar/StudySidebar.tsx (hunk 1 above)
Comment (MatthewBemis) [reply ui-admin/src/navbar/StudySidebar.tsx:127]: fwiw, this section used to be called `Administration` before i removed it in the redesign

also good catch with the differing casing- looks like Mailing list differs too

**Pull request merged**